internal/utils/tag.go

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/fatih/structtag"
+)
+
+func RemoveTagsOmitempty(s interface{}, key string) (interface{}, error) {
+	var value reflect.Value
+	if reflect.TypeOf(s).Kind() == reflect.Ptr {
+		value = reflect.Indirect(reflect.ValueOf(s))
+	} else {
+		value = reflect.ValueOf(s)
+	}
+	t := value.Type()
+	sf := make([]reflect.StructField, 0)
```

Comment:
You can initialize slice with `t.NumField()` initial size, and put items using `sf[i] = field` in loop to avoid resizing on grow